README.md

```diff
@@ -177,14 +177,24 @@ python end_to_end.py
 ```
 
 
+
 * ### Server And Client
+
 It is simple client server communication script, will add more functionality in future.
 ```bash
 cd server_client
 python client.py
 python server.py
 ```
 
+
+### Get_Lyrics

```

Comment:
add bullet point like this 
![selection_021](https://user-images.githubusercontent.com/10944610/31702300-c0501f58-b3f3-11e7-922c-06c26d20ad32.png)
 by adding `*` in front of line.